.gitignore

```diff
@@ -1,7 +1,19 @@
-development.log
-commands.log
+# Logs
+*.log
+
+# Extras
 extra/lgit.rb
+
+# Notes
 notes/go.notes
 TODO.notes
 TODO.md
+
+# Tests
 test/repos/repo
+
+# Binaries
+lazygit
+
+# Dependencies
+vendor/*
```

Comment:
I don't think it's a good idea to ignore the vendor directory, see [here](https://golang.github.io/dep/docs/FAQ.html#should-i-commit-my-vendor-directory). Updating dependencies should be a conscious decision.

Reply:
That's true, it's indeed a pro/con thingy. Let's see what the others think, idm either options.

Reply:
happy to merge the other changes to the .gitignore file though

Reply:
Good points, i'll take this PR down and create a new one without the vendor packages! @jesseduffield @remyabel